### PR TITLE
Add Strengths Content module.

### PR DESCRIPTION
**Pull request by eschlange**
* Added migration for new model.
* Added association for new model.
* Added model validation spec.
* Updated schema.
* Added new and index views for strengths.
* Added content providers and modules for strengths.
* Added seed data for strength related objects.

[#110760028] [#110760060]

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -558,6 +558,16 @@
   add_index "social_networking_shared_items", ["created_at"], name: "index_social_networking_shared_items_on_created_at", using: :btree
   add_index "social_networking_shared_items", ["participant_id"], name: "index_social_networking_shared_items_on_participant_id", using: :btree
 
+  create_table "strengths", force: :cascade do |t|
+    t.integer  "participant_id"
```
Comment (michaelwehrley) [inline db/schema.rb]: Is there a check here for `null`?  I am not sure if you need to augment the migration to include `null: false`.  At least I would add a check in the spec file: `spec/models/strength_spec.rb`

Comment (eschlange) [inline db/schema.rb]: Good catch. Made the update to the spec.

Comment (michaelwehrley) [inline db/schema.rb]: I was wondering if the same is in kindness journal? https://github.com/NU-CBITS/marigold/blob/master/db/migrate/20160328185659_create_kindness.rb#L4  Can you make the same update?  Thanks.

Comment (eschlange) [inline db/schema.rb]: Done. :sunglasses:

Comment (michaelwehrley) [inline db/schema.rb]: you the man!  Thank you!

Comment (michaelwehrley) [inline db/schema.rb]: Is that at the database level?

Comment (eschlange) [inline db/schema.rb]: I accidentally deleted a parentheses  before my last commit. It should be fixed with my latest commit.

**Comment by michaelwehrley** [comment]
:+1: :shaved_ice:

**Pull request merged**